src/tactic/interactive_expr.lean

```diff
@@ -153,6 +161,12 @@ meta def view {γ} (tooltip_component : tc subexpr (action γ)) (click_address :
     on_click (λ _, action.on_click ca),
     key s
   ] [html.of_string s]]
+| ca (sf.block i a) := do
+  inner ← view ca a,
+  pure [h "span" [cn "indent-code", style [("--indent-level", to_string i)]] inner]
+| ca (sf.highlight c a) := do
+  inner ← view ca a,
+  pure [h "span" [cn $ "highlight_" ++ c.to_string] inner]
```

Comment:
Can you try replacing `[cn $ "highlight_" ++ c.to_string]` with `[cn c.to_string]`? There are already some CSS classes which style particular colours.

Reply:
I guess a problem is that you want the highlighting colours to be theme-dependent so that they are readable for both light and dark themes.

Reply:
> BTW, would it be possible to inline the `.indent-code` class for now (i.e. use `style` instead of `cn`)? Then we could test it without modifying the extension.

Unfortunately not. We need to change the CSS in the extension in such a way, that the whole subgoals become `display:inline-block` as opposed to `display:inline` what they are now. If you do it only on one tag, it completely moves stuff around and it's not even in the same order anymore.

I was in contact with @EdAyers about this and understood, that we can take a look at the vscode extension shortly, this is mostly preparatory work, really.